dev/snippets/lib/main.dart

```diff
@@ -99,6 +108,11 @@ void main(List<String> argList) {
       .firstWhere((SnippetType type) => getEnumName(type) == args[_kTypeOption], orElse: () => null);
   assert(snippetType != null, "Unable to find '${args[_kTypeOption]}' in SnippetType enum.");
 
+  if (args[_kShowDartPad] == true && snippetType != SnippetType.application) {
+    errorExit('$snippetType was selected, but the --dartpad flag is only valid '
```

Comment:
```suggestion
    errorExit('--type=${describeEnum(snippetType)} was specified, but the --dartpad flag is only valid '
```

You might have to steal `describeEnum` from `foundation/diagnostics.dart`, since we're not importing Flutter code here.

Reply:
Another good catch. I ended up going with `${args[_kTypeOption]}`, since that's the approach you use a few lines above.

Reply:
Yeah, that's fine too.